record_store.gemspec

```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'google-cloud-dns'
   spec.add_runtime_dependency 'ruby-limiter', '~> 1.0', '>= 1.0.1'
   spec.add_runtime_dependency 'ns1'
+  spec.add_runtime_dependency 'oci'
```

Comment:
🤔 what if they release a version of the `oci` gem that has a completely different interface from the one that `record_store` depends on?  Consider a semver range, similar to the ones for dependencies like `fog`, `dnsimple` and `ruby-limiter`.